### ISSUE TITLE
imagenet-console custom DIGITS caffe model "could not parse layer type Python"

### ISSUE THREAD

**Issue by DennisFaucher** [dusty-nv/jetson-inference#629]
Hello,

I just trained the KITTI cars data in DIGITS from the jetson-inference tutorial and downloaded and extracted the model into a "cars" directory.

When I run this command, I get the above error. Any ideas? I saw a closed issue about removing the python section from the end of deploy.prototxt but that just resulted in a different error about softmax. How does one download and use a new DIGITS model successfully? Thank you.

jetson-inference/build/aarch64/bin/imagenet-console ./NYC_Cars.jpg  output_0.jpg --prototxt=$NET/deploy.prototxt --model=$NET/snapshot_iter_4800.caffemodel --labels=$NET/labels.txt --input_blob=data --output_blob=softmax

Full output:

jetson-inference/build/aarch64/bin/imagenet-console ./NYC_Cars.jpg  output_0.jpg --prototxt=$NET/deploy.prototxt --model=$NET/snapshot_iter_4800.caffemodel --labels=$NET/labels.txt --input_blob=data --output_blob=softmax

imageNet -- loading classification network model from:
         -- prototxt     cars/deploy.prototxt
         -- model        cars/snapshot_iter_4800.caffemodel
         -- class_labels cars/labels.txt
         -- input_blob   'data'
         -- output_blob  'softmax'
         -- batch_size   1

[TRT]   TensorRT version 7.1.0
[TRT]   loading NVIDIA plugins...
[TRT]   Plugin creator registration succeeded - ::GridAnchor_TRT
[TRT]   Plugin creator registration succeeded - ::NMS_TRT
[TRT]   Plugin creator registration succeeded - ::Reorg_TRT
[TRT]   Plugin creator registration succeeded - ::Region_TRT
[TRT]   Plugin creator registration succeeded - ::Clip_TRT
[TRT]   Plugin creator registration succeeded - ::LReLU_TRT
[TRT]   Plugin creator registration succeeded - ::PriorBox_TRT
[TRT]   Plugin creator registration succeeded - ::Normalize_TRT
[TRT]   Plugin creator registration succeeded - ::RPROI_TRT
[TRT]   Plugin creator registration succeeded - ::BatchedNMS_TRT
[TRT]   Could not register plugin creator:  ::FlattenConcat_TRT
[TRT]   Plugin creator registration succeeded - ::CropAndResize
[TRT]   Plugin creator registration succeeded - ::DetectionLayer_TRT
[TRT]   Plugin creator registration succeeded - ::Proposal
[TRT]   Plugin creator registration succeeded - ::ProposalLayer_TRT
[TRT]   Plugin creator registration succeeded - ::PyramidROIAlign_TRT
[TRT]   Plugin creator registration succeeded - ::ResizeNearest_TRT
[TRT]   Plugin creator registration succeeded - ::Split
[TRT]   Plugin creator registration succeeded - ::SpecialSlice_TRT
[TRT]   Plugin creator registration succeeded - ::InstanceNormalization_TRT
[TRT]   completed loading NVIDIA plugins.
[TRT]   detected model format - caffe  (extension '.caffemodel')
[TRT]   desired precision specified for GPU: FASTEST
[TRT]   requested fasted precision for device GPU without providing valid calibrator, disabling INT8
[TRT]   native precisions detected for GPU:  FP32, FP16, INT8
[TRT]   selecting fastest native precision for GPU:  FP16
[TRT]   attempting to open engine cache file cars/snapshot_iter_4800.caffemodel.1.1.7100.GPU.FP16.engine
[TRT]   cache file not found, profiling network model on device GPU
[TRT]   device GPU, loading cars/deploy.prototxt cars/snapshot_iter_4800.caffemodel
could not parse layer type Python
[TRT]   device GPU, failed to parse caffe network
[TRT]   device GPU, failed to load cars/snapshot_iter_4800.caffemodel
[TRT]   failed to load cars/snapshot_iter_4800.caffemodel
[TRT]   imageNet -- failed to initialize.
imagenet-console:   failed to initialize imageNet

**Comment by DennisFaucher** [comment]
UPDATE
Just as a test, I performed a manual install of DIGITS 6.1.1 on an Ubuntu 16.04 VM (The last recipe that worked for me). Built the standard cat-dog image recognition model. Downloaded the model to my Xavier and imagenet-console worked like a charm. There is some inconsistency between the models created in the Ubuntu 18.04 AWS DIGITS Marketplace image and jetson-inference.
